examples/tests/valid/nil.w

```diff
@@ -3,7 +3,7 @@ bring cloud;
 class Foo {
   inflight var optionalVar: str?;
   
-  init() {
+  inflight init() {
```

Comment:
@marciocadev, @hasanaburayyan - curious why the `inflight` keyword was added? Correct me if I'm wrong, but I think it wasn't necessary (but not harmful too) since we can init inflight fields also  in preflight init, right?

Reply:
@staycoolcall911 from what I understood from issue #2582, inflight fields should be initialized in inflight initializers, but my understanding may be wrong. However, if that's the case, I don't understand why the initializer could receive the "inflight" prefix.

Reply:
I might be wrong as well :)
My understanding is that inflight fields could be initialized in either preflight or inflight init.
They must be already initialized in inflight after init ended.

Reply:
We aren't supposed to be able to access inflight members (fields or methods) in preflight. So I think @marciocadev is correct here.